README.md

```diff
@@ -1,6 +1,15 @@
 # Fake-AirPlay
 This crate creates a fake AirPlay service, which will be discoverable by apple devices on the network.
 ## Usage
+> ❗ To use fake-airplay make sure you have the **avahi daemon** running ❗
+
+To start the **avahi daemon** install the `nss-mdns` package and run:
```

Comment:
This isn't necessarily true. On arch it's the avahi package and on debian it's avahi-daemon. We can trust the users to figure it out themselves.